feconf.py

```diff
@@ -33,6 +33,10 @@
 IS_MINIFIED = os.environ.get('MINIFICATION') == 'True'
 
 # Whether we should serve the development or production experience.
+# This is environment specific variable(DEV_MODE) changes to true only
```

Comment:
ooh ok like [adding numpy](https://github.com/oppia/oppia/blob/develop/appengine_config.py#L88) in development mode, clearing [cookies](https://github.com/oppia/oppia/blob/develop/core/controllers/base.py#L145), set the dev mode green label that appears only in dev_mode ... I think only those.